### PR TITLE
Fix shu-mutou's info

### PR DESCRIPTION
**Pull request by shu-mutou**
I have belonged only to NEC.

### CODE REVIEW

File: developers_affiliations2.txt [hunk 1]
```diff
@@ -21200,8 +21200,7 @@ shterman: roysh!mellanox.com
 shtripat: shtripat!redhat.com, shtripat!users.noreply.github.com, shubhendu.tripathi!gmail.com
 	Red Hat
 shu-mutou: shu-mutou!rf.jp.nec.com, shu-mutou!users.noreply.github.com, shu.mutow!gmail.com
```
Comment (oomichi) [inline developers_affiliations2.txt:21202]: LGTM as `shu-mutou!rf.jp.nec.com` which is NEC mail domain.

Comment (oomichi) [reply developers_affiliations2.txt:21202]: /cc @oomichi

**Comment by lukaszgryglicki** [comment]
Let me know when i can merge this.

**Comment by shu-mutou** [comment]
ASAP, thanks a lot!!

**Pull request merged**